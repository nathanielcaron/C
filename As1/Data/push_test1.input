u 10
x