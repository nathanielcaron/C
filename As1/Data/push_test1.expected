10
Successfully executed 1 instructions
|_ 10 _|
1 elements
